lego/apps/achievements/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+
+from lego.apps.users.models import User
+from lego.utils.models import BasisModel
+
+from .constants import ACHIEVEMENT_IDENTIFIERS
+
+
+class Achievement(BasisModel):
+    identifier = models.CharField(choices=ACHIEVEMENT_IDENTIFIERS, max_length=128)
+    hidden = models.BooleanField(default=False, null=False, blank=False)
+    user = models.ForeignKey(
+        User, related_name="achievements", on_delete=models.CASCADE
+    )
+    level = models.PositiveSmallIntegerField(default=0)
+
+    @property
+    def percentage(self):
+        total_users = User.objects.count() or 1
+        achievement_users = (
+            Achievement.objects.filter(identifier=self.identifier, level=self.level)
```

Comment:
I think this should also count people who have a higher level of the same achievement

Reply:
```suggestion
            Achievement.objects.filter(identifier=self.identifier, level__gte=self.level)
```
I think something like this would work?